### DISCUSSION TITLE
comments_xtd_default_followup

### DISCUSSION THREAD

**Discussion by ogurec-ogurec** [danirus/django-comments-xtd#288] [Q&A]
https://github.com/danirus/django-comments-xtd/issues/206#issuecomment-778579019

> Can you run your project's shell command and run this code?
> 
> ```
> >>> from django_comments_xtd import get_form, get_version
> >>> get_version()
> '2.8.3'
> >>> get_form()
> django_comments_xtd.forms.XtdCommentForm
> ```
> 
> What do they return? If the version is older than [2.8.3](https://github.com/danirus/django-comments-xtd/releases/tag/2.8.3), the setting was not implemented yet. It's only available as of 2.8.3. If on the other hand the form is a class other than `django_comments_xtd.forms.XtdCommentForm`, the class you would be using might implement the followup field without taking into account the new setting.
> 
> Let us know.

Did everything as you said, here is the result:
```
>>> from django_comments_xtd import get_form, get_version
>>> get_version()
'2.8.3'
>>> get_form()
<class 'django_comments_xtd.forms.XtdCommentForm'>
>>>
```
Notify "on follow-up comments" float is not enabled by default. Maybe a problem somewhere on the frontend?

**Comment by ogurec-ogurec** [comment]
my `form.py` is clearly different from this commit:

https://github.com/drholera/django-comments-xtd/commit/fcc2f2ecc6f3d88065ba95a664f617abd00c3074

i using update pip install django-comments-xtd -U

**Reply by danirus** [reply]
The problem is that `COMMENTS_XTD_DEFAULT_FOLLOWUP` is ignored by the JavaScript plugin yet.
Just reported it: https://github.com/danirus/django-comments-xtd/issues/291
